tests/waku_archive_legacy/test_driver_postgres.nim

```diff
@@ -4,11 +4,13 @@ import std/[sequtils, options], testutils/unittests, chronos
 import
   waku/waku_archive_legacy,
   waku/waku_archive_legacy/driver/postgres_driver,
+  waku/waku_archive/driver/postgres_driver as new_postgres_driver,
```

Comment:
Me personally better like to tag the legacy one with alias instead of the new. Due it will ease the work of removing.

Reply:
> Me personally better like to tag the legacy one with alias instead of the new. Due it will ease the work of removing.

Yes good point! We will be able to completely remove the whole `tests/waku_archive_legacy/test_driver_postgres.nim` file in the future